backend/src/config/mod.rs

```diff
@@ -568,3 +733,7 @@ pub fn configure_rec<'a, Db: DbHandle>(
     }
     .boxed()
 }
+#[instrument]
+pub fn not_found() -> Error {
```

Comment:
Should this be part of the db things? I don't know where this should live, or if it should.